samples/drivers/watchdog/README.rst

```diff
@@ -0,0 +1,57 @@
+.. _watchdog sample:
+
+Watchdog (WD)
```

Comment:
I'm not sure if `WD` is the common abbreviation for watchdogs. The Zephyr APIs use `wdt_` or `WDT_` prefixes. I suggest to leave it out in this title.
```suggestion
Watchdog Sample
```